.github/workflows/python-main.yml

```
@@ -1,917 +1,33 @@
-    strategy:
-      matrix:
-        python-set: ["cp39-cp39", "cp310-cp310", "cp311-cp311", "cp312-cp312", "cp313-cp313", "cp314-cp314"]
-    env:
-      DEPTHAI_BUILD_BASALT: ON
-      DEPTHAI_BUILD_PCL: ON
-      DEPTHAI_BUILD_RTABMAP: ON
-      DEPTHAI_BUILD_KOMPUTE: ON
-      VCPKG_BINARY_SOURCES: "clear;files,/home/runner/.vcpkg,readwrite"
     steps:
       - name: Cache vcpkg folder
         uses: actions/cache@v3
```

actionlint (1.7.12)

[error] the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)